### ISSUE TITLE
Track Continuation

### ISSUE THREAD

**Issue by kabirnagpal** [kabirnagpal/ML-Track#15]
We want to take the repository forward to 14 Weeks that will roughly account for 100 Days.

- Deep Learning Code shall be included

- NLP processing and code shall be included

- Project links need to be included.

**Comment by kabirnagpal** [comment]
@tarushi98 @l-ightmare @radioactive11 @subhankar01 @Ramitphi
Please discuss the curriculum.

**Comment by tarushi98** [comment]
I feel in deep learningl we should cover 5 topics and give each topic two weeks . 
For eg: if one topic is introduction to neural networks , in which we cover what they are, how to lay the basic structure in tf, activation functions , loss functions , etc. We can distribute this in two weeks. 
We can make one topic be NLP in this. This will acct for 10 weeks

This will give us 4 weeks for projects , in which we can cover projects like recommendation sys. If we want to give one week to each project then we will have to look for four projects.

**Comment by kabirnagpal** [comment]
@tarushi98 Please give a detail of weeks like

- Week 9: 
- Week 10: 

Like that

**Comment by tarushi98** [comment]
@kabirnagpal @radioactive11 @subhankar01 @Ramitphi @l-ightmare 
Kabir , discussed an idea for the Deep Learning track. Check it out and let's discuss what all can be added.
We have thought of covering it this way:
1) Explaining few basics by coding the Dog Cat classifier in either Keras or fastai or both
2)Discussing  preprocessing methods for NLP and implementing one Application.
3) One implementation each of LSTM and RNN.
These all can be covered for 2 weeks each.
@kabirnagpal If I forgot something, please add.

**Comment by kabirnagpal** [comment]
1. Week 9: NLP processing methods
2. Week 10: Deep Learning ( explanation of concepts using a simple classifier in Keras or FastAI )
3. Week 11: NLP using Deep Learning ( intro to RNN )

Please reply here on the thread and recommend ideas and changes
@tarushi98 @radioactive11 @subhankar01 @l-ightmare @Ramitphi

**Comment by kabirnagpal** [comment]
@charansoneji

**Comment by kabirnagpal** [comment]
1. week 9: Open CV and Yolo @l-ightmare @radioactive11 
2. week 10: MNIST, Cat Dog @charansoneji @subhankar01 @Ramitphi
3. week 11: Transfer Learning, RNN: COVID @tarushi98 @kabirnagpal 
4. Week 12: NLP: preprocessing, glove, summarisation
5. week 13: Embedding layers, FASTAI, Bert: optional
6. week 14: Gans
7. week 15: deployment: fast API

Project: Face recognition, Recommendation systems, Neural Style
@tarushi98 @radioactive11 @subhankar01 @Ramitphi @charansoneji @l-ightmare
